monitoring/mock_uss/msgsigning/routes_msgsigning.py

```diff
@@ -0,0 +1,18 @@
+import flask
+
+from monitoring.mock_uss import webapp, config
+from loguru import logger
+import os
+
+
+@webapp.route(
+    "/mock/msgsigning/.well-known/uas-traffic-management/pub.der", methods=["GET"]
+)
+def get_public_key():
+    public_key_file_location = os.path.join(
+        webapp.config.get(config.KEY_CERT_BASE_PATH), "messagesigning/mock_faa_pub.der"
+    )
```

Comment:
If the comment above is adopted, this could be changed to:

```suggestion
    public_key_file_location = os.path.join(
        webapp.config.get(config.KEY_CERT_BASE_PATH),
        db.value.public_key_name
    )
```

...and this line would also be needed:

```python
from monitoring.mock_uss.msgsigning.database import db
```